tletools/tle.py

```diff
@@ -162,7 +161,8 @@ def a(self):
     def nu(self):
         """True anomaly."""
         if self._nu is None:
-            self._nu = _M_to_nu(self.M * DEG2RAD, self.ecc) * RAD2DEG
+            M = self.M * DEG2RAD
```

Comment:
```suggestion
            M = ((self.M + 180) % 360 - 180) * DEG2RAD
```

> Fields 2.3, 2.4, 2.6, and 2.7 all have units of degrees and can range from 0 up to 360 degrees

https://celestrak.com/columns/v04n03/#FAQ01

see https://github.com/FedericoStra/tletools/pull/6#issuecomment-628876160 by @tomaszmrugalski

Reply:
The fix @astrojuanlu proposed works. I figured it out on my own, but Juan was faster :) I've created #7 (which has cherry-picked this change + Juan's fix + extra unit-tests)